x-pack/plugins/security_solution/public/cases/pages/index.tsx

```diff
@@ -100,6 +100,7 @@ const CaseContainerComponent: React.FC = () => {
           owner: [APP_ID],
           features: {
             metrics: ['alerts.count', 'alerts.users', 'alerts.hosts', 'connectors', 'lifespan'],
+            alerts: { isExperimental: true },
```

Comment:
@elastic/security-threat-hunting-explore I put it as experimental because to my knowledge it will become GA on 8.5. The actions are missing in the cases alert table for the security solution so it makes sense to me to have it as experimental.

Reply:
LGTM. But need to confirm with @paulewing

Reply:
@YulNaumenko Any news on this?